src/components/Infoblock/Infoblock.tsx

```diff
@@ -0,0 +1,183 @@
+                )}
+              </Stack>
+              {(ctaPrimary || ctaSecondary) && (
+                <Stack direction="row" spacing={2}>
+                  {ctaPrimary && (
+                    <Button
+                      aria-label={ctaPrimary.title}
+                      variant="contained"
+                      href={ctaPrimary.href}
+                    >
+                      {ctaPrimary.label}
+                    </Button>
+                  )}
+                  {ctaSecondary && (
+                    <Button
+                      aria-label={ctaSecondary.title}
+                      variant="outlined"
+                      href={ctaSecondary.href}
+                    >
+                      {ctaSecondary.label}
+                    </Button>
+                  )}
+                </Stack>
+              )}
+            </Stack>
+          </Box>
+          <Box
+            gridColumn={{
+              xs: "span 6",
+              md: inverse ? "2 / span 5" : "7 / span 5",
+            }}
+            gridRow={{
+              xs: "auto",
+              md: 1,
+            }}
+            mr={{
+              xs: "0",
+              md: imageShadow ? "43px" : "0",
+            }}
+            my="auto"
+          >
+            <Box
+              sx={{
+                display: "grid",
+                maxHeight: aspectRatio === "4/3" ? "450px" : "none",
+                maxWidth: aspectRatio === "4/3" ? "none" : "270px",
+                mx: "auto",
+              }}
+            >
+              {imageShadow && (
+                <Box
+                  sx={{
+                    display: {
+                      xs: "none",
+                      md: "block",
+                    },
+                    gridArea: "1 / 1 / 2 / 2",
+                    zIndex: 2,
+                    backgroundImage: `url(${imageTypePattern})`,
+                    backgroundSize: "cover",
+                    height: "100%",
+                    width: "100%",
+                    maxWidth: aspectRatio === "4/3" ? "600px" : "270px",
+                    maxHeight: aspectRatio === "4/3" ? "450px" : "480px",
+                    mt: aspectRatio === "4/3" ? "16%" : "25%",
+                  }}
+                />
+              )}
+              <Box
+                component="img"
+                alt={altText}
+                src={image}
+                sx={{
+                  objectFit: "cover",
+                  objectPosition: "50%",
+                  maxWidth: {
+                    xs: "100%",
+                    md: aspectRatio === "4/3" ? "600px" : "270px",
+                  },
+                  maxHeight: {
+                    xs: "300px",
+                    md: aspectRatio === "4/3" ? "450px" : "480px",
+                  },
+                  width: "100%",
+                  height: "100%",
+                  gridArea: "1 / 1 / 2 / 2",
+                  zIndex: 5,
+                  aspectRatio,
```

Comment:
You're using native `aspect-ratio` property here. That's perfectly fine if you browse the page with the latest browser releases, but we should set a CSS property fallback (using padding-bottom hack) to give proper support for older versions.